firefox-ios/Storage/Rust/RustPlaces.swift

```diff
@@ -666,7 +666,10 @@ extension RustPlaces {
                     // as the title
                     title = info.url
                 }
-                let site = Site(url: info.url, title: title)
+                // FIXME: FXIOS-10740 Workaround for an iOS 18 HistoryPanel crash with diffable data sources until we can
+                // get a unique GUID from application services (should be sent in the payload with synced history items)
+                let hashValue = "\(info.url)_\(info.timestamp)".hashValue
```

Comment:
If we REALLY want to be confident this is unique, we should use `let hashValue = UUID().uuidString.hashValue`...